assets/xml/objects/object_osn.xml

```diff
@@ -113,5 +113,6 @@
         <Skeleton Name="object_osn_Skel_0202F0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_OSN_LIMB_NONE" LimbMax="OBJECT_OSN_LIMB_MAX" EnumName="object_osn_Limbs" Offset="0x202F0" />
         <Animation Name="object_osn_Anim_020530" Offset="0x20530" />
         <!-- <Blob Name="object_osn_Blob_020540" Size="0x20" Offset="0x20540" /> -->
+        <Skeleton Name="gDekuMaskSkel" Type="Flex" LimbType="Standard" Offset="0x20550" />
```

Comment:
```suggestion
        <Skeleton Name="gDekuMaskSkel" Type="Flex" LimbType="Standard" Offset="0x20550" />
```
Hmm, is that skeleton header the whole blob? If so can we remove it from the xml?

Reply:
There doesn't appear to be an unaccounted for array, so I'll just delete the blob comment? If I'm understanding what you're saying right